apps/ensadmin/src/hooks/use-connection-url-param.tsx

```diff
@@ -59,7 +59,11 @@ export function useRawConnectionUrlParam(): UseRawConnectionUrlParamResult {
   const searchParams = useSearchParams();
 
   // Get the current raw connection URL param
-  const rawConnectionUrlParam = searchParams.get(RAW_CONNECTION_PARAM_KEY);
+  // Use useMemo to prevent unnecessary re-renders
+  const rawConnectionUrlParam = useMemo(
```

Comment:
not sure the usememo is doing anything here?

Reply:
Reverted. There was a moment during hydration where the param wasn't present, and then was added, and then readded during my testing which was causing an infinite loop. But I can't seem to recreate this now so reverting the change.